lib/ssh/test/ssh_options_SUITE.erl

```diff
@@ -778,6 +780,47 @@ connectfun_disconnectfun_server(Config) ->
 	    {fail, "No connectfun action"}
     end.
 
+%%--------------------------------------------------------------------
+bannerfun_server(Config) ->
```

Comment:
could you also add an interop verification in ssh_to_openssh_SUITE?
maybe it would be enough to adjust existing testcase, so it uses Erlang server sending a banner and OpenSSH client ...

`ssh_to_openssh_SUITE:eserver_oclient_renegotiate_helper1/` - maybe enough to modify this function?

Reply:
Yes. This is what I was after. thanks!